vipps-qr-api-webhooks.md

```diff
@@ -0,0 +1,38 @@
+<!-- START_METADATA
+---
+title: QR API Webhooks
+sidebar_label: Webhooks
+sidebar_position: 35
+description: Find information on the use of webhooks in the QR solution
+pagination_prev: Null
+pagination_next: Null
+---
+END_METADATA -->
+
+# QR Api Webhooks
+The QR solution currently has one webhook a merchant can subscribe to.
+
+The central part of the Merchant Callback QR flow is the callback, that will be sent to the merchant when a customer scans the QR.
+To receive these callbacks, the merchant needs to register a subscription to the *user.checkin.v1* event. 
+
+Here is the payload for the *user.checkin.v1* event:
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| customerToken | base64 string | A reference to the customer. Should be used when initiating a payment through the ePayments API |
+| merchantQrId | string | The id of the QR code that has been scanned. Defined by the merchant when the QR was created |
+| merchantSerialNumber | string | A unique id of the sales unit that the QR scanned belongs to |
+| initiatedAt | Timestamp in ISO 8601 format | The timestamp of when the customer scanned the QR |
```

Comment:
Is this timestamp in UTC or with a Time Offset?
According to the example below it seems to be UTC so you could specify that in the description if you want to :)